Chef/Cookbooks/Relativity/recipes/post_relativity_check_correct_relativity_version_is_installed.rb

```diff
@@ -0,0 +1,28 @@
+custom_log 'custom_log' do msg 'Starting Checking Correct Relativity Version is Installed' end
+    start_time = DateTime.now
+    custom_log 'custom_log' do msg "recipe_start_time(#{recipe_name}): #{start_time}" end
+    
+    rap_file_path = win_friendly_path(File.join(Chef::Config[:file_cache_path], '\cookbooks\Relativity\files\default\RA_Smoke_Test_2.2.2.1.rap'))
+    
+    # Install Checking Correct Relativity Version is Installed
+    custom_log 'custom_log' do msg 'Checking Correct Relativity Version is Installed' end
+    
+    powershell_script 'checking_correct_relativity_version_is_installed' do
+      code <<-EOH
+        #{node['powershell_module']['import_module']}
+        $relativityAndInvariantVersions = [System.IO.File]::ReadAllText(#{node['relativity_and_invariant']['response_file']['version_file']})
+        $indexOfComma = $relativityAndInvariantVersions.IndexOf(',')
+        $indexOfColon = $relativityAndInvariantVersions.IndexOf(':')
+        $installerRelativityVersion = $relativityAndInvariantVersions.Substring(($indexOfColon + 2), ($indexOfComma - ($indexOfColon + 2)))
+        $installerRelativityVersion = $installerRelativityVersion.Replace(" ", "")
+        Show-InstallerAndInstanceRelativityVersionAreEqual -RelativityInstanceName #{node['windows']['new_computer_name']} -RelativityAdminUserName #{node['relativity']['admin']['login']} -RelativityAdminPassword #{node['relativity']['admin']['password']} -SqlAdminUserName #{node['sql']['user']['eddsdbo']['login']} -SqlAdminPassword #{node['sql']['user']['sa']['password']} -InstallerRelativityVersion $installerRelativityVersion 
```

Comment:
_Confirm_ seems like a correct verb instead of _Show_. Please refer to the approved verb list here - https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7